vignettes/geologic-map.Rmd

```diff
@@ -39,46 +39,46 @@ install.packages("devtools")
 devtools::install_github("palaeoverse/rmacrostrat")
 ```
 
-You can now load `rmacrostrat` using the default `library` function:
+You can now load `rmacrostrat` using the standard `library` function:
 
 ```{r load-library, eval = TRUE}
```

Comment:
For the record, `eval` is already set to `TRUE` in the `setup` chunk, so you don't _have_ to include it in all of the chunks.